tests/unit/seed/embed/test_bootstrap_link_via_app_data.py

```diff
@@ -110,17 +110,18 @@ def test_seed_link_via_app_data(tmp_path, coverage_env, current_fastest, copies)
 @pytest.fixture()
 def read_only_app_data(temp_app_data):
     temp_app_data.mkdir()
+    orig_mode = os.stat(str(temp_app_data)).st_mode
     try:
-        os.chmod(str(temp_app_data), S_IREAD | S_IRGRP | S_IROTH)
+        os.chmod(str(temp_app_data), S_IRUSR | S_IXUSR | S_IRGRP | S_IROTH)
```

Comment:
S_IREAD should have been S_IRUSR -- I also added S_IXUSR to aid in debugging (execute is necessary on directories to `ls` them)